tests/test_no_extra_install.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests Sklearn GradientBoostingClassifier converters.
```

Comment:
This is a typo. Maybe something like `Test Hummingbird when no extra dependencies are installed`.

Reply:
good catch